hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-resourcemanager/src/test/java/org/apache/hadoop/yarn/server/resourcemanager/webapp/TestWebServiceUtil.java

```diff
@@ -213,15 +207,74 @@ public static void assertJsonResponse(ClientResponse response,
       IOException {
     assertJsonType(response);
     JSONObject json = response.getEntity(JSONObject.class);
+    sortQueuesLexically(json);
```

Comment:
I was meant to ask this before: is there a reason we're using two separate Json libraries to do (in the end) a string comparison? Couldn't we move completely to Jackson?

Reply:
Moved to jackson. There was a change in the test data, because jettison converted the floats (e.g. capacity=100.0) to an int (capacity=100).